lib/node-utils/index.ts

```diff
@@ -510,3 +533,115 @@ export function hasImportMatch(
 
   return importNode.local.name === identifierName;
 }
+
+export function getStatementCallExpression(
+  statement: TSESTree.Statement
+): TSESTree.CallExpression | undefined {
+  if (isExpressionStatement(statement)) {
+    const { expression } = statement;
+    if (isCallExpression(expression)) {
+      return expression;
+    }
+
+    if (
+      ASTUtils.isAwaitExpression(expression) &&
+      isCallExpression(expression.argument)
+    ) {
+      return expression.argument;
+    }
+
+    if (isAssignmentExpression(expression)) {
+      if (isCallExpression(expression.right)) {
+        return expression.right;
+      }
+
+      if (
+        ASTUtils.isAwaitExpression(expression.right) &&
+        isCallExpression(expression.right.argument)
+      ) {
+        return expression.right.argument;
+      }
+    }
```

Comment:
@zaicevas these are the new checks for the cases you mentioned. There must be a better way of finding those calls in the tree, this seems a little bit brittle but I guess it works for now.